src/test/java/com/synacy/lesson01/exercises/ComplexNumberTest.java

```diff
@@ -0,0 +1,59 @@
+package com.synacy.lesson01.exercises;
+
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+
+public class ComplexNumberTest {
+    ComplexNumber complexNumber1;
+    ComplexNumber complexNumber2;
+    ComplexNumber expectedComplexNumber;
+
+    @Before
+    public void setup(){
+        complexNumber1 = new ComplexNumber(5.0,3.0);
+        complexNumber2 = new ComplexNumber(2.0,2.0);
+    }
+    @Test
+    public void add_shouldGetExpectedComplexNumberReal() throws Exception {
+        expectedComplexNumber = new ComplexNumber(7.0,5.0);
+        ComplexNumber actualComplexNumber = complexNumber1.add(complexNumber2);
+        assertEquals(expectedComplexNumber.getReal(), actualComplexNumber.getReal(),0.00001);
+    }
+    @Test
+    public void add_shouldGetExpectedComplexNumberImaginary() throws Exception {
```

Comment:
The `ComplexNumber` is a concept here. In this case, it would be okay (and preferable) if we have combined this and the previouse test case into one